src/users/user.php

```diff
@@ -156,14 +156,23 @@ public function get()
 	{
 		$input = JFactory::getApplication()->input;
 		$id = $input->get('id', 0, 'int');
+		$xidentifier	= $input->server->get('HTTP_X_IDENTIFIER', '', 'String');
 
 		/*
 		 * If we have an id try to fetch the user
 		 * @TODO write user field mapping logic here
 		 */
 		if ($id)
 		{
-			$user = JUser::getInstance($id);
+			if ($xidentifier)
```

Comment:
This if/else is not needed. Check `retriveUser` it is already taking into account id/email/username.

Reply:
@coolbung done changes